examples/basics/plotting/grid_two_row_two_col_.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# vispy: gallery 30
+# -----------------------------------------------------------------------------
+# Copyright (c) 2015, Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+"""
++-----+-----------+
+| 1   |     2     |
++-----+----+------+
+|   3      |   4  |
++----------+------+
```

Comment:
I don't think this layout should be possible with two rows and two cols. Two rows and *three* cols, yes. But with two cols only, it implies the width of the column changes from row 0 to row 1, which is wrong.